### ISSUE TITLE
[2024.04] Lyrics API stopped working

### ISSUE THREAD

**Issue by raitonoberu** [raitonoberu/sptlrx#43]
Well, it wasn't hard to expect, Spotify got a change on the server (over 2 years btw!) that broke our lyrics scraping. I'm currently looking into the problem, and thankfully it seems easily fixable.

But I'm very busy these days, so expect changes in the range of a day to a couple weeks.

Thanks for staying here ❤️

**Comment by raitonoberu** [comment]
I spent an hour trying to figure out why the same query works in curl and doesn't work in Go. Turns out Spotify servers deny access to requests with Go's [default user agent](https://go.dev/src/net/http/request.go#L508), but not curl :)

If you are using [lyricsapi](https://lyricsapi.vercel.app/api/lyrics?name=nirvana) as your lyrics provider (player in config is not set to spotify & cookie is empty), this should already work. If you're using your own cookie, you'll have to wait until I complete #44 because putting it off is no longer acceptable.
